src/huntsman/pocs/utils/pyro/service.py

```diff
@@ -1,23 +1,38 @@
 import time
-from Pyro5.api import Daemon as PyroDaemon
+from multiprocessing.context import Process
 
+from Pyro5.api import Daemon as PyroDaemon
 from panoptes.utils.library import load_module
 from panoptes.utils.config.client import get_config, server_is_running
-
 from huntsman.pocs.utils.logger import logger
 from huntsman.pocs.utils.pyro.nameserver import get_running_nameserver
 from huntsman.pocs.utils.config import get_own_ip
 
 
+def pyro_service_process(auto_start=False, *args, **kwargs):
```

Comment:
We seemed to be hitting weird `Segmentation faults` on the pis when running pyro services inside Processes. Not sure exactly what the reason was though, but removing code similar to this resolved the problem. Just an FYI in case we see the same thing again!

Reply:
But it should be in a docker container on the pi right? Not sure how the Pi would know the different. But yeah, odd.